_posts/2015-06-12-topics.md

```diff
@@ -871,3 +878,238 @@ This topic is deprecated. Please use [shipment:packed](#shipment_packed) instead
     }
 }
 ```
+<h3 id="tracker_updated">
+    tracker:updated
+</h3>
+
+```json
+{
+    "topic" : "tracker:updated",
+    "message" : {
+        "order_ref" : "9099",
+        "order_unique_id" : "100000018",
+        "shipment_unique_id" : "100000007",
+        "warehouse_id": 1,
```

Comment:
Add `tracker_id` to all `tracker:*` webhooks since we will have a tracker API where the user can lookup trackers by id.